### PR TITLE
Update Hat Sample Pack Link + Added Mods Checklist

### PR DESCRIPTION
**Pull request by Tatohead**
Update Hat Sample Pack Download Host Link with a Basic Checklist

The check list acts as simplified instructions and easy to remind the users what step might have been missing

### CODE REVIEW

**Comment by Tatohead** [comment]
Checklist example:
---

If your mod isn't working please look at the check list to see what  step might be missing:

Check list:
[✓] Latest version of Desktop Goose
[✓] mod dll is in mods folder within its own folder
[✓] config file has EnableMods=True
[✓] Save the config
[✓] Restart the program

**Comment by Tatohead** [comment]
I made it into an FAQ Section, that has a check list. Is that fine?

**Comment by VukAnd** [comment]
I was referring to the FAQ section in the readme [here](https://github.com/DesktopGooseUnofficial/tree/master/README.md#faq).

**Comment by Tatohead** [comment]
Oh oK I'll edit it again

**Comment by Tatohead** [comment]
So does that look fine?

File: README.md [hunk 1]
```diff
@@ -71,3 +71,17 @@ There are no modpacks available right now!
 ### Okay I got a mod, placed it in the right folder and all that but it's not working, how do I enable it?
 
 ![](https://i.imgur.com/ZNB4wsy.gif)
+
+### If your mod isn't working please look at the check list to see what  step might be missing:
```
Comment (VukAnd) [inline README.md]: Why are there random spaces here? Here's a fix:
```suggestion
### If your mod isn't working please look at the checklist to see what step might be missing:
```

File: README.md [hunk 2]
```diff
@@ -71,3 +71,17 @@ There are no modpacks available right now!
 ### Okay I got a mod, placed it in the right folder and all that but it's not working, how do I enable it?
 
 ![](https://i.imgur.com/ZNB4wsy.gif)
+
+### If your mod isn't working please look at the check list to see what  step might be missing:
+
+**Check list:**
```
Comment (VukAnd) [inline README.md]: Again, why is there a space here?
```suggestion
**Checklist:**
```

File: README.md [hunk 3]
```diff
@@ -71,3 +71,17 @@ There are no modpacks available right now!
...
 
 ![](https://i.imgur.com/ZNB4wsy.gif)
+
+### If your mod isn't working please look at the check list to see what  step might be missing:
+
+**Check list:**
+
+[   ] Latest version of Desktop Goose
+
+[   ] mod dll is in mods folder within its own folder
```
Comment (VukAnd) [inline README.md]:
```suggestion
[   ] Mod DLL is in mods folder within its own folder
```

File: README.md [hunk 4]
```diff
@@ -71,3 +71,17 @@ There are no modpacks available right now!
...
+
+[   ] config file has EnableMods=True
```
Comment (VukAnd) [inline README.md]:
```suggestion
[   ] Config file has EnableMods=True
```

**Pull request merged**